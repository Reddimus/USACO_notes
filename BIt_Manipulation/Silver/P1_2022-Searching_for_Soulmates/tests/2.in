10
49236 23945
17761 47420
52480 54434
22705 19565
89539 4818
25510 19063
4658 57974
61152 48153
71305 51287
36800 96727
