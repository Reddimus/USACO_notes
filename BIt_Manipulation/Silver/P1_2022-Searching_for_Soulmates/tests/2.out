42
38
25
40
37
38
37
36
42
37
